rb/lib/selenium/webdriver/bidi/session.rb

```diff
@@ -32,6 +32,28 @@ def status
           Status.new(status['ready'], status['message'])
         end
 
+        def subscribe(events, browsing_contexts = nil)
+          events_list = events.instance_of?(Array) ? events : [events]
```

Comment:
You can write it in a more compact way:

```suggestion
          events_list = Array(events)
```